README.md

```diff
@@ -1 +1,39 @@
-# release
\ No newline at end of file
+# Release
+This repository will maintain the releases for the aurae project.
+
+### Project Status
+
+The project is very young and under active development. The APIs are subject to change without notice until further notice.
+As we continue to develop the project the APIs will stabilize and eventually a long term stable release will be offered.
+
+At this time the project should not be run in production.
+
+Please read [getting involved](https://aurae.io/community/#getting-involved) if you are interested in joining the project in its early phases. Contribution types of all types and ranges are welcome.
+
+
+### Release Artifacts
+
+The following artifacts will be available with every stable build. Artifacts are intended to be used with the same anything sharing the same minor version. `v0.1.2` of `ae` would work with `v0.1.4` of `aurae`, but not `v0.2.1`.
```

Comment:
how do grpc protos fit into this? should we version them by minor version?  if so, what do we do when we bump the major version?

Reply:
For now they'll match the versions being created elsewhere, until we're out of the initial experimental builds